src/tox.ini

```
@@ -80,7 +80,7 @@ deps =
     pytest-cov
 
 [testenv:func]
-commands = pytest {toxinidir}/tests/functional {posargs:-v}
+commands = pytest {toxinidir}/tests/functional -vvvv
```

It's really useful to be able to change parameters, e.g. `-k test_config_changed`.
```suggestion
commands = pytest {toxinidir}/tests/functional {posargs:-vvvv}
```